xchangelog

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# xchangelog PKGNAME - open package changelog
+
+XBPS_DISTDIR=$(xdistdir) || exit 1
+
+pkg=$1
+[ -z "$pkg" ] && pkg=$(basename "$PWD")
+
+if [ -f "$pkg" ]; then
+	template="$pkg"
+elif [ -f "$pkg/template" ]; then
+	template="$pkg/template"
+elif [ -f "$XBPS_DISTDIR/srcpkgs/$pkg/template" ]; then
+	template="$XBPS_DISTDIR/srcpkgs/$pkg/template"
+else
+	exit 1
+fi
+
+. "$template"
+
+if [ -z "$changelog" ]; then
+	echo "No changelog defined in template"
+	exit 1
+fi
+
+if curl -sI -- "$changelog" | grep -i "^content-Type: text/plain;"; then
```

Comment:
don't match for `;` here

Reply:
I did that to make sure that it matched only `text/plain` and not, say, `text/plainfoo`, but it's not a big deal I guess.